fw/http_parser.c

```diff
@@ -7532,6 +7605,38 @@ __h2_req_parse_content_length(TfwHttpMsg *msg, unsigned char *data, size_t len,
 	return ret >= 0 ? CSTR_NEQ : ret;
 }
 
+int
+h2_set_hdr_content_type(TfwHttpReq *req, const TfwCachedHeaderState *cstate)
+{
+	int r;
+
+	if (cstate->is_set) {
```

Comment:
IIRC the Linux kernel coding style also suggests early returns:

```suggestion
	if (!cstate->is_set)
		return 0;
```
and probably we can avoid some line breaks.